modern/src/components/reports/ReportNavbar.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { AppBar, Toolbar, Typography, IconButton } from '@material-ui/core';
+import MenuIcon from '@material-ui/icons/Menu';
+import t from '../../common/localization';
+
+const ReportNavbar = ({ openDrawer, setOpenDrawer, reportTitle }) => {
+
+  return (
+    <AppBar position="fixed" color="inherit">
+      <Toolbar>
+        <IconButton
+          color="inherit"
+          aria-label="open drawer"
+          edge="start"
+          onClick={() => setOpenDrawer(!openDrawer)}>
```

Comment:
This can probably just be a callback. You don't need to pass `openDrawer`.

Reply:
Ok. Removed openDrawer.